root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -3942,7 +3942,7 @@ _.each(testData, function (subtest, i) {
       const validationResults = _.reduce(LINK_TYPES[relationshipType],
         function (results, relUuid, entityType) {
           const rule = validationRules[relUuid];
-          const isValid = rule ? rule(cleanUrl) || false : true;
+          const isValid = rule ? rule(cleanUrl).result || false : true;
```

Comment:
Maybe type could be checked so as not to have to change all of the returned results without message.

Reply:
That would introduce inconsistency though. I'd much rather everything followed a consistent style.

Reply:
Which “inconsistency”?

Reply:
Some rule results being an object and some being a Boolean, and us having to change them for adding messages rather than just adding error to the object. If some will be objects then I wouldn't want others not to be. I will split this in several commits later though :)